pom.xml

```diff
@@ -55,12 +69,46 @@
                     <artifactId>spring-boot-maven-plugin</artifactId>
                     <version>${spring.version}</version>
                 </plugin>
-                <plugin>
-                    <groupId>com.google.cloud.tools</groupId>
-                    <artifactId>jib-maven-plugin</artifactId>
-                    <version>${jib.version}</version>
-                </plugin>
             </plugins>
         </pluginManagement>
     </build>
+
+    <profiles>
+        <profile>
+            <id>docker</id>
```

Comment:
Put this under o profile, since we might not want to always generate the images.
One image was fine, but now the build is getting slow with 5 images.

In order to generate images now, the profile should be used

mvn clean install -Pdocker